io-classes/src/Control/Monad/Class/MonadMVar.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE DefaultSignatures      #-}
 {-# LANGUAGE QuantifiedConstraints  #-}
 {-# LANGUAGE TypeFamilyDependencies #-}
+{-# LANGUAGE TypeOperators          #-}
```

Comment:
Why this addition?

Reply:
`ghc-9.4` warns about using `~` without `TypeOperators` enabled.